### ISSUE TITLE
Unsupported Git Flow version

### ISSUE THREAD

**Issue by SKProCH** [OpherV/gitflow4idea#302]
* **I'm submitting a ...**
  - [x] bug report

* **Is this a bug? Sorry about that. If so give me explicit details how to reproduce:**

1. Open Rider
2. ![image](https://user-images.githubusercontent.com/29896317/81143638-f6f11c00-8f7a-11ea-8b50-c5ef7782364d.png)
3. Profit

  - Gitflow4idea version: 0.7.2
  - Gitflow version: 1.12.3 (AVH Edition)
  
  - JetBrains Rider 2020.1.1
Build #RD-201.6668.201, built on April 28, 2020
Runtime version: 11.0.6+8-b765.25 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
.NET Framework 4.0.30319.42000
Windows 10 10.0

This message began to appear after updating the plugin on the latest version.
![image](https://user-images.githubusercontent.com/29896317/81143696-17b97180-8f7b-11ea-8f72-e2dc48b82585.png)

**Comment by driici** [comment]
HI. same with PhpStorm 2020.1.1 Build #PS-201.7223.96, built on April 30, 2020
Also it works on some of my repositories (all with same origin server, different creation dates). I have also tried to clone repo to new directory; however it didnt help either.

**Comment by OpherV** [comment]
@SKProCH @driici  can you check if it's the same issue like #299 ?

**Comment by driici** [comment]
> @SKProCH @driici can you check if it's the same issue like #299 ?
copying misc.xml from other project wokerd for me. Sorry, did not find that isssue earlier. But from content of misc.xml there is no specific configuration for my current project and for git flow neither. I think at least change of message will be in order, since old CLI is different case, than missing some XML; PHPStorm creates some xml files in .idea only if they are needed. So if there is nothing to write to misc.xml it does not exists.

**Comment by OpherV** [comment]
@driici oh it's definitely a bug - it shouldn't happen at all :)
I'll try and tackle it for next version but in the meanwhile this is a solution so that you're not stuck.

**Comment by driici** [comment]
@OpherV  Thank you very much!

**Comment by targzeta** [comment]
I have the same problem. The file .idea/misc.xml doesn't exists and I have no idea what to write to.

If I simply "touch" the file, the plugin shows 'No Gitflow'. The 'Init Repo' option do nothing.

Best regards,
Emanuele

**Comment by OpherV** [comment]
@targzeta remove this:
```
[gitflow "branch"]
        master = master
        develop = develop
[gitflow "prefix"]
        feature = feature/
        release = release/
        hotfix = hotfix/
        support = support/
        versiontag = 
```

Re-init gitflow. It should sort it out

**Comment by SKProCH** [comment]
Yes, for me this problem is identical to that described in #299.
I copied misc.xml from the newly created project. But why is this happening and how to fix it automatically?

**Comment by targzeta** [comment]
> @targzeta remove this:
> 
> ```
> [gitflow "branch"]
>         master = master
>         develop = develop
> [gitflow "prefix"]
>         feature = feature/
>         release = release/
>         hotfix = hotfix/
>         support = support/
>         versiontag = 
> ```
> 
> Re-init gitflow. It should sort it out

It works if:
- I don't have any [gitflow] directives whitin .git/config file
- I create and fill the .idea/misc.xml with:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project version="4" />
```

Is it possible to make it works also without misc.xml file?

Thank you in advance,
Emanuele

**Comment by OpherV** [comment]
@targzeta of course. A fix is planned for next version

**Comment by OpherV** [comment]
Fix in 0.7.3 now out
